perma_web/fabfile/dev.py

```diff
@@ -120,6 +123,19 @@ def init_db():
     local("python manage.py migrate --database=perma-cdxline")
     local("python manage.py loaddata fixtures/sites.json fixtures/users.json fixtures/folders.json")
 
+@task
+def reset_hard_db():
+    """
+        Drops the perma and perma_cdxline databases and creates and inits them again
+        Let folks run this if they're not in Django's debug mode
+    """
+
+    if not settings.DEBUG and not confirm("WARNING! You're about to drop the Perma.cc DBs. Continue anyway?"):
```

Comment:
Hmm. I think this switch is maybe backwards? "If not settings.DEBUG" means "if in production", right?